apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TestDatabaseFixture.cs

```diff
@@ -21,15 +21,15 @@ namespace Google.Cloud.Spanner.Data.IntegrationTests
 {
     public class TestDatabaseFixture : IDisposable {
 
-        private readonly string _databaseName = "t_" + Guid.NewGuid().ToString("N").Substring(0, 28);
+        private readonly string _databaseName = "scratch";// "t_" + Guid.NewGuid().ToString("N").Substring(0, 28);
         private Task _creationTask;
         private readonly string _testTable = "TestTable";
         private DatabaseAdminClient _databaseAdminClient;
 
         public void Dispose() {
-            if (_databaseAdminClient != null) {
-                _databaseAdminClient.DropDatabase(new DatabaseName(TestProjectName, TestInstanceName, DatabaseName));
-            }
+            //if (_databaseAdminClient != null) {
+            //    _databaseAdminClient.DropDatabase(new DatabaseName(TestProjectName, TestInstanceName, DatabaseName));
+            //}
```

Comment:
Are these meant to be commented out? Should there be a TODO here to uncomment them at some point?